pkg/lib/config/saml.go

```diff
@@ -19,7 +19,9 @@ var _ = Schema.Add("SAMLServiceProviderConfig", `
 	"type": "object",
 	"additionalProperties": false,
 	"properties": {
-		"id": { "type": "string" }
+		"id": { "type": "string" },
+		"name_id_format": { "$ref": "#/$defs/SAMLNameIDFormat" },
```

Comment:
NameID should be `nameid`

Reply:
The PascelCase NameID should be name_id in snake_case?

Reply:
Since it is `rn:oasis:names:tc:SAML:1.1:nameid-format:unspecified`, I think `nameid` is one word.

Reply:
I see. Ok let me change it.